src/RadhydroShockCGS/test_radhydro_shock_cgs.cpp

```diff
@@ -68,6 +68,7 @@ template <> struct quokka::EOS_Traits<ShockProblem> {
 	static constexpr double mean_molecular_weight = m_H;
 	static constexpr double boltzmann_constant = k_B;
 	static constexpr double gamma = gamma_gas;
+	static constexpr double hydrogen_mass_code_units = 1.0 / C::n_A;
```

Comment:
This should be set equal to the atomic mass unit.

Reply:
yes this is on my TODO list